templates/includes/messages.html

```diff
@@ -1,9 +1,14 @@
+{% load static %}
+
+{% block css %}
+<link rel="stylesheet" href="{% static 'css/allauthstyles.css' %}" />
+<link rel="stylesheet" href="{% static 'css/profile.css' %}" />
+{% endblock %}
+
 {% if messages %}
 <div>
     {% for message in messages %}
-        <h1>{{ message }}</h1>
-        <!-- TO DO : help wanted -->
-        <small>TO DO : templates > includes > messages.html (styles)</small>
+        <h3>{{ message }}</h3>

```

Comment:
I would change the styling on this and use a boostrap alert class maybe. Also, `h3` seems a bit big.

Reply:
Haven't yet gotten around to fully customizing the messages yet. Will probably look to setup specific alert classes depending on the type of message. For now have reduced size to <h5>.